subjects/get-alpha/README.md

```diff
@@ -0,0 +1,23 @@
+## getalpha
+
+### Instructions
+
+Write  a program that takes an `int` [0-127] as an argument and returns the corresponding ASCII table character, followed by newline.
+
+- If the integer above 127 or less than 0 returns newline.
+- If it's not a number returns newline.
+- If the number of arguments is above 1 return newline.
```

Comment:
Needs to be fixed. "...is above 1  return a newline `\n`."

Reply:
fixed